kayenta-judge/src/main/scala/com/netflix/kayenta/judge/classifiers/metric/MannWhitneyClassifier.scala

```diff
@@ -128,12 +128,17 @@ class MannWhitneyClassifier(tolerance: Double=0.25,
                         experiment: Metric,
                         direction: MetricDirection,
                         nanStrategy: NaNStrategy,
-                        isCriticalMetric: Boolean): MetricClassification = {
+                        isCriticalMetric: Boolean,
+                        isDataRequired: Boolean): MetricClassification = {
 
     //Check if there is no-data for the experiment or control
     if (experiment.values.isEmpty || control.values.isEmpty) {
       if (nanStrategy == NaNStrategy.Remove) {
         val reason = s"Missing data for ${experiment.name}"
+        // if the config indicates that the given metric should have data but not critically fail the canary.
```

Comment:
Missing the word 'Check' from the beginning of the comment (to match the comments above and below).

Could also swap the order of `!isCriticalMetric` and `isDataRequired` in the conditional to match the ordering in the comment.

Reply:
- [X] fixed